tests/unit/moduleapi/crash.tcl

```diff
@@ -34,18 +38,25 @@ if {!$::valgrind} {
             set res [wait_for_log_messages 0 {"*=== REDIS BUG REPORT START: Cut & paste starting from here ===*"} 0 10 1000]
             set loglines [lindex $res 1]
 
-            set res [wait_for_log_messages 0 {"*Crashed running the instruction at*"} $loglines 10 1000]
-            set loglines [lindex $res 1]
+            if {$backtrace_supported} {
+                set res [wait_for_log_messages 0 {"*Crashed running the instruction at*"} $loglines 10 1000]
+                set loglines [lindex $res 1]
 
-            set res [wait_for_log_messages 0 {"*Crashed running signal handler. Providing reduced version of recursive crash report*"} $loglines 10 1000]
-            set loglines [lindex $res 1]
-            set res [wait_for_log_messages 0 {"*Crashed running the instruction at*"} $loglines 10 1000]
-            set loglines [lindex $res 1]
+                set res [wait_for_log_messages 0 {"*Crashed running signal handler. Providing reduced version of recursive crash report*"} $loglines 10 1000]
+                set loglines [lindex $res 1]
+                set res [wait_for_log_messages 0 {"*Crashed running the instruction at*"} $loglines 10 1000]
+                set loglines [lindex $res 1]
```

Comment:
@madolson why did you make this block conditional?

Reply:
"Crashed running the instruction at" is also in the ifdef for having a backtrace. See https://github.com/redis/redis/blob/14e4a9835af2bb7df8c42cfb850bd19708cd73d0/src/debug.c#L2264